ogr/ogrsf_frmts/arrow_common/ograrrowlayer.hpp

```diff
@@ -4286,6 +4286,17 @@ inline int OGRArrowLayer::GetArrowSchema(struct ArrowArrayStream *stream,
 /*                     GetArrowSchemaInternal()                         */
 /************************************************************************/
 
+static bool IsSilentlyIgnoredFormatForGetArrowSchemaArray(const char *format)
+{
+    // n: null
+    for (const char *pszSilentyIgnoredFormat : {"n"})
```

Comment:
Why a loop?  The `pszSilentyIgnoredFormat` could probably be static const anyway.

Reply:
was a provision if we added support for other formats, but I'll simplify this